README.md

```diff
@@ -1,177 +1,178 @@
+## Directory scanning
+
+To scan a directory, add the following step:
 
-For example: 
 ```yaml
- - name: Scan image
-   uses: anchore/scan-action@v1
+ - name: Scan current project
+   uses: anchore/scan-action@v2
    with:
-     image-reference: "localbuild/testimage:latest"
-     fail-build: true
+     path: "."
 ```
 
-As a result of the action, you'll see some files in the `anchore-reports` directory in the workspace:
+The `path` key allows any valid path for the current project. The root of the path (`"."` in this example) is the repository root.
 
-* `policy_evaluation.json` - Default anchore policy evaluation of the image
-* `vulnerabilities.json` - Vulnerabilities found in the image
-* `content.json` - packages (rpms, debs, npms, jars, gems, etc) found in the image, including versions, locations, and licenses
+## Failing a build on vulnerability severity
+To have the build step fail in cases where there are vulnerabilities with a specific severity level, then set the `fail-build` to `true`. By default, the severity level is `Medium`, but this can be adjusted using the `severity-cutoff` field.
 
-### Scanning Application Packages and OS Packages in the Container
-
-By default, the action will only match vulnerabilities against OS/distro packages (rpms, dpkg, apk, etc). This is done
-to allow the use of a much smaller scan image and thus faster scans. However, Anchore has the ability to match vulnerabilities
-against npms, gems, python pip packages, and java (jars, wars, etc) as well. This scan will take longer, but produce a more
-holistic view of the container vulnerability set. To enable this feature, set the 'include-app-packages' input parameter to 'true'.
-
-For example:
 ```yaml
- - uses: anchore/scan-action@v1
-       with:
-         image-reference: "localbuild/testimage:latest"
-         dockerfile-path: "Dockerfile"
-         fail-build: true
-         include-app-packages: true
+ - name: Scan image
+   uses: anchore/scan-action@v2
+   with:
+     image: "localbuild/testimage:latest"
+     fail-build: true
 ```
 
-### Supplying a Custom Policy
-
-The default policy the action applies can fail the build if vulnerability with severity >= High is found that has a fix available.
-To have other behavior, you can provide your own policy using the `custom-policy-path` parameter. The path is considered relative
-to the root of the workspace (which gets reset to the repository if you use the `checkout` action).
+With a different severity level:
 
-For example, to include a custom policy as: .anchore/policy.json in your code repository, set:
 ```yaml
- - uses: anchore/scan-action@v1
-       with:
-         image-reference: "localbuild/testimage:latest"
-         dockerfile-path: "Dockerfile"
-         fail-build: true
-         custom-policy-path: .anchore/policy.json
+ - name: Scan image
+   uses: anchore/scan-action@v2
+   with:
+     image: "localbuild/testimage:latest"
+     fail-build: true
+     severity-cutoff: Critical
```

Comment:
Is this value case sensitive? (Capitalizing "Critical" makes it look like it is)

Reply:
It is

Reply:
Okay, gotcha. I could see this being something that trips users up if they don't realize that case matters here. What are your thoughts? Would it be okay for it to be case insensitive? If we leave it case sensitive and a user makes a mistake, will it be obvious to them exactly what to change?

Reply:
Agreed, it would be good to remove all case-sensitivity here, but can document it (comment in example perhaps?) since it isn't a change from previous. The can relax that in future patch update.

Reply:
I can make it case-insensitive, but @zhill is right, this is a no-change from before. It isn't obvious today if they get the casing incorrect.

Reply:
This is now case insensitive, and the docs and workflows have been updated to reflect that